src/ensembl/src/content/app/entity-viewer/gene-view/components/protein-image/ProteinImage.tsx

```diff
@@ -42,7 +42,8 @@ const ProteinImage = (props: ProteinImageProps) => {
     .range([0, props.width])
     .clamp(true);
 
-  const midStyles = classNames(transcriptsListStyles.middle, styles.middle);
+  const midStyles = transcriptsListStyles.middle;
+  const rightStyles = classNames(transcriptsListStyles.right, styles.label);
```

Comment:
Would it be better to rename these to something like `trackContainerStyles` and `labelStyles`?

Reply:
I don't feel strongly one way or the other. Renamed them if you prefer these names.